shared/src/commonMain/kotlin/org/hyperskill/app/home/presentation/HomeFeature.kt

```diff
@@ -41,14 +34,12 @@ interface HomeFeature {
          * @property problemOfDayState Problem of the day state.
          * @property repetitionsState Topics repetitions state.
          * @property isRefreshing A boolean flag that indicates about is pull-to-refresh is ongoing.
-         * @property isLoadingMagicLink A boolean flag that indicates about magic link loading.
          * @see Streak
          * @see ProblemOfDayState
          */
         data class Content(
             val problemOfDayState: ProblemOfDayState,
             val repetitionsState: RepetitionsState,
-            val isFreemiumEnabled: Boolean,
```

Comment:
`isFreemiumEnabled` We are using this flag on platforms for displaying "Solve unlimited" badge on the problem of day card

Reply:
Now we have a bug:

```kotlin
is Message.HomeSuccess ->
    state.copy(
        homeState = HomeState.Content(
            message.problemOfDayState, // problemOfDayState
            message.repetitionsState, // repetitionsState
            message.isFreemiumEnabled // isRefreshing
        )
    ) to emptySet()
```

Reply:
https://github.com/hyperskill/mobile-app/pull/707/commits/efb782cb052206e3aa4247d5d022a80097a1164b

Reply:
cc @vladkash @XanderZhu